homeassistant/components/wyoming/satellite.py

```diff
@@ -544,3 +552,38 @@ async def _stt_stream(self) -> AsyncGenerator[bytes, None]:
                 yield chunk
         except asyncio.CancelledError:
             pass  # ignore
+
+    @callback
+    def _handle_timer(
+        self, event_type: intent.TimerEventType, timer: intent.TimerInfo
+    ) -> None:
+        """Forward timer events to satellite."""
+        assert self._client is not None
+
+        _LOGGER.debug("Timer event: type=%s, info=%s", event_type, timer)
+        event: Event | None = None
+        if event_type == intent.TimerEventType.STARTED:
+            event = TimerStarted(
+                id=timer.id,
+                total_seconds=timer.seconds,
+                name=timer.name,
+                start_hours=timer.start_hours,
+                start_minutes=timer.start_minutes,
+                start_seconds=timer.start_seconds,
+            ).event()
+        elif event_type == intent.TimerEventType.UPDATED:
+            event = TimerUpdated(
+                id=timer.id,
+                is_active=timer.is_active,
+                total_seconds=timer.seconds,
+            ).event()
+        elif event_type == intent.TimerEventType.CANCELLED:
+            event = TimerCancelled(id=timer.id).event()
+        elif event_type == intent.TimerEventType.FINISHED:
+            event = TimerFinished(id=timer.id).event()
+
+        if event is not None:
+            # Send timer event to satellite
+            self.hass.async_create_background_task(
```

Comment:
call this on the config entry. `self.config_entry.async_create_background_task(`